backend/src/main/scala/cromwell/backend/BackendJobExecutionActor.scala

```diff
@@ -54,5 +53,5 @@ trait BackendJobExecutionActor extends BackendJobLifecycleActor with ActorLoggin
   /**
     * Abort a running job.
     */
-  def abortJob: Future[JobAbortResponse]
+  def abortJob: Unit
```

Comment:
Couldn't it still be useful to have some kind of response from that actually ? Say you try to abort a JES job, but for some reason there's an error while requesting the abort to JES (the jib can't be found or whatever..) Then you'll never know about this error :/